README.md

```diff
@@ -1,25 +1,82 @@
-# `corppa`  PPA full-text corpus utilities
+# corppa
+
+This repository is research software developed as part of the [Ends of Prosody](https://cdh.princeton.edu/projects/the-ends-of-prosody/), which is associated with the [Princeton Prosody Archive](https://prosody.princeton.edu/) (PPA). This software is particularly focused on research and work related to PPA full-text and page image corpora.
+
+> [!WARNING]
```

Comment:
Trying out the format I found here https://github.com/orgs/community/discussions/16925

Reply:
Is there a place we should document this for future reference?

Reply:
the syntax? maybe we could add a link to some team documentation somewhere...

Reply:
Yeah, that sounds good to me. This seems like a nice feature to use.